android/src/main/java/com/swmansion/reanimated/NodesManager.java

```diff
@@ -398,6 +398,13 @@ private void handleEvent(Event event) {
       int viewTag = event.getViewTag();
       String key = viewTag + eventName;
       EventNode node = mEventMapping.get(key);
+      // If for some reason name resolution failed try a more generic approach
+      // converting `topEventName` to `onEventName`
+      if (node == null && eventName != null && eventName.startsWith("top")) {
+        eventName = "on" + eventName.substring(3);
+        key = viewTag + eventName;
+        node = mEventMapping.get(key);
+      }
```

Comment:
Shouldn't `.resolveCustomEventName()` take care of that? If events are registered in `ModuleConstants` then they should be converted. Looks like a hacky way to do it.

Reply:
You are right - this is a bit hacky. This was an effort to make `TextInput` events work with reanimated if I recall correctly. I guess submitting a PR to `react-native` is the right approach. I think it was `onBlur` event that wasn't supported because it was exported as `topBlur`

Reply:
I can revert commit 76b0dd1.

Reply:
If it doesn't break anything I'd like to get this removed.

Reply:
Done  0a9c204
FYI `TextInput` doesn't export event names properly. See [reference](https://github.com/facebook/react-native/blob/065fbe3be53e156fe4f2dbf47791be53b340096f/ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java#L151).